server/api/index.js

```diff
@@ -38,6 +38,15 @@ router.get('/pull-request-review-requests', (req, res, next) => {
     .catch(next)
 });
 
+router.get('/pull-request-review-requests/:week', (req, res, next) => {
```

Comment:
since we're going to utilize the `loadMetricsForWeek` action to get data for the existing metrics page as week as this new timeline, I think we can remove this new route and modify `/metrics/:week`if we need to